labs/instana/3-agent-installation/index.mdx

```diff
@@ -212,27 +212,62 @@ you are not logged on already, ssh to the **control node**.
 
 To install Instana agent on OpenShift cluster you need to authenticate to the
 OCP cluster with appropriate permissions (cluster admin role). In the lab
-environment, the **control node** is already logged into the cluster. But, we
-need to make sure you are logged into the **Application** cluster and not the
+environment, the **control node** is already logged into the cluster and the
+user group has the necessary permissions. But, we need to make sure you are
+logged into the **Application** cluster and not the
 **AIOps** cluster. Issue the following command:
 
 ```sh
 oc get projects
 ```
 
 Then, look for the **qotd** and **qotd-load** projects. If you don't see those
-two projects, then you must login to the correct cluster. Open the OpenShift
-user interface for the **Application** cluster. Then, click in the upper right
-corner on the username and select **Copy loging command**
+two projects, then you must login to the correct cluster. Check the current
+context with the command:
 
-![](images/ocLogin.png)
+``` sh
+oc config current-context
+```
+
+The output should be similar to:
+
+``` text
+tech-jam-dal10-04-manage-to/ccljdtjd00i4gbvqat00/admin
+```
+
+where in according to the Lab stand architecture we have to use cluster
+tech-jam-dal10-04-**manage-to** as the **Application** cluster for Instana
+agent installation.
+
+In case, if the current cluster is different, then we need to connect to
+**Application** cluster, for that check the name of the correct cluster 
+it should have "to" prefix in it's name:
+
+``` sh
+oc config get-contexts
+```
+
+copy correct name of the cluster:
 
-Then, click on **Display Token**
+``` text
+CURRENT   NAME                                                       CLUSTER                                              AUTHINFO                     NAMESPACE
+          tech-jam-dal10-04-manage-from/ccljdu9d0tn57qnqat0g/admin   tech-jam-dal10-04-manage-from/ccljdu9d0tn57qnqat0g   admin/ccljdu9d0tn57qnqat0g   default
```

Comment:
To avoid confusion can you please update the names 'manage-to' and 'manage-from' to their new equivalents, for example 'lon06-05-apps' and 'lon06-05-aiops'.